core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -321,6 +325,48 @@ protected function compile()
 					$objTemplate->hasContext = true;
 				}
 
+				// Add image to search results
+				$objTemplate->hasImage = false;
+				if (isset($arrResult[$i]['meta']) && null !== $arrResult[$i]['meta'])
+				{
+					$meta = json_decode($arrResult[$i]['meta'], true);
+					// Reverse so the last element wins
+					$meta = array_reverse($meta);
```

Comment:
Not sure if this is correct? One has to win for this case but is it the last or the first appearance on the page? /cc @ausi

Reply:
How about an images array?

Reply:
I don't think there's any value in that. It's named `primaryImageOfPage`. There can only ever be one primary image. And moving it to the template would only delegate the problem to the user. They would pick the first array element anyway.

Reply:
I think we should choose the first one then.

Reply:
Done.